src/components/dialogs/pick-request-type.vue

```diff
@@ -45,7 +45,7 @@ import { RequestActionsI } from '@/interfaces'
 import { ActionBindingIF } from '@/interfaces/store-interfaces'
 import { RequestActions } from '@/list-data'
 
-/*
+/**
```

Comment:
This is now a JSDoc comment, which displays when you hover over the component/class.